### DISCUSSION TITLE
v2022.0.2

### DISCUSSION THREAD

**Discussion by brettcannon** [brettcannon/WWBD#27] [Announcements]
## What's Changed
* Remove progress reporting by @brettcannon in https://github.com/brettcannon/WWBD/pull/24
* Stop using `PYTHONUTF8` by @brettcannon in https://github.com/brettcannon/WWBD/pull/25

**Full Changelog**: https://github.com/brettcannon/WWBD/compare/v2022.0.1...v2022.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/brettcannon/WWBD/releases/tag/v2022.0.2'>v2022.0.2</a>.</em>

**Comment by PaulFWatts** [comment]
I've only just come across this and tried it for the first time.

Worked perfectly on ubuinu 22.04 (I already had the prequsites installed)

Saves a lot of fooling around.

I suggest that this evolves into the main Python extension and allows the creation of a Python Project  with a Python Project Explorer similar to how you can now create a Java Project with a Java Project Explorer.

Vscode just keeps getting better and better :-)

**Reply by brettcannon** [reply]
The plan is anything that works here will be integrated into the core Python extension.

As for a project explorer, are you referring to https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency ? And if so, can you open a separate idea for this and explain what you're looking for?
